Few-iOS/Button.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  Button.swift
+//  Few
+//
+//  Created by Coen Wessels on 13/03/15.
+//  Copyright (c) 2015 Josh Abernathy. All rights reserved.
+//
+
+import UIKit
+
+private class FewButton: UIButton {
```

Comment:
Nevermind, i can fix this :satisfied:. If i set the `trampoline` property of the `Button` to the value of the old `Button` it will remain to work after a rerender.